apps/nextjs-website/src/components/PressRelease.tsx

```diff
@@ -25,12 +25,14 @@ const makeTextSectionProps = ({
   subtitle,
   body,
   date,
+  backlink,
   ...rest
 }: PressReleaseSection &
   Omit<SiteWidePageData, 'themeVariant'>): PressReleaseProps => ({
   ...(subtitle && { subtitle }),
   body: MarkdownRenderer({ markdown: body, locale, defaultLocale }),
   date: formatDateToLocale(date, locale),
+  ...(backlink && { backlink }),
```

Comment:
```suggestion
  ...(backlink && { backlink: {
    label: backlink.label,
    href: localizeURL({
      URL: backlink.href,
      locale,
      defaultLocale,
    }),
  }}),
```
Remember to localize URLs